libraries/AP_Scripting/examples/LED_roll.lua

```diff
@@ -25,7 +25,8 @@ chan = chan + 1
 gcs:send_text(6, string.format("LEDs: chan=" .. tostring(chan)))
 
 -- initialisation code
-serialLED:set_num_LEDs(chan,  num_leds)
+--serialLED:set_num_neopixel(chan,  num_leds)
```

Comment:
I guess we should remove this bit of debug?

Reply:
This is just to make it easy for people who have neopixles to change the script, you just have to comment out the ProfiLED one and swap in the neopixel